eras/shelley/impl/src/Cardano/Ledger/Chain.hs

```diff
@@ -62,17 +62,13 @@ instance NoThunks ChainPredicateFailure
 
 chainChecks ::
   MonadError ChainPredicateFailure m =>
-  Version ->
   ChainChecksPParams ->
   BHeaderView c ->
   m ()
-chainChecks maxpv ccd bhv = do
```

Comment:
Could you please revert changes to this function and all other changes that depended on it